readme.md

```diff
@@ -24,8 +24,11 @@ The ucf-resource-search shortcode has several options:
 
 ## Changelog ##
 
-### 1.0.1 ###
+
+### 1.0.2 ###
 * Fixed PHP notices
+* Removed Athena classes
```

Comment:
There are still a good number of athena-specific classes left in the shortcode markup, e.g. `.btn` classes, `.form-group`, `.form-control`--was the intent to remove athena references entirely, or just the copied-over css rules included in the plugin?

Reply:
I removed the form-* classes. I left in the btn* classes without the btn* css it doesn't look as nice, but it does work. I added the range logic. Thanks, Jim.